mappings/net/minecraft/item/CompassItem.mapping

```diff
@@ -1,5 +1,16 @@
 CLASS baq net/minecraft/item/CompassItem
 	CLASS baq$1
+		FIELD b angle D
+		FIELD c step D
+		FIELD d lastTick J
+		METHOD a getYaw (Latf;)D
+			ARG 1 entity
+		METHOD a getAngle (Lbhi;D)D
+			ARG 1 world
+			ARG 2 entityYaw
+		METHOD a getSpawnArctangent (Lbhj;Laii;)D
```

Comment:
Recall this was brought up on Discord, but not sure where that went. `getAngleToSpawn`?

Reply:
Resolved by 54d9bb3